lib/ansible/modules/packaging/os/sorcery.py

```diff
@@ -0,0 +1,632 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+---
+module: sorcery
+short_description: Package manager for Source Mage GNU/Linux
+description:
+    - Manages "spells" on Source Mage GNU/Linux using I(sorcery) toolchain
+author: "Vlad Glagolev (@vaygr)"
+version_added: "2.3"
+notes:
+    - When all three components are selected, the update goes by the sequence --
+      Sorcery -> Grimoire(s) -> Spell(s); you cannot override it.
+    - grimoire handling (i.e. add/remove, including SCM/rsync versions) is not
+      yet supported.
+requirements:
+    - bash
+options:
+    name:
+        description:
+            - Name of the spell
+            - multiple names can be given, separated by commas
+            - special value '*' in conjunction with states C(latest) or
+              C(rebuild) will update or rebuild the whole system respectively
+        required: false
+        aliases: ["spell"]
+        default: null
+
+    state:
+        description:
+            - Whether to cast, dispel or rebuild a package
+            - state C(cast) is an equivalent of C(present), not C(latest)
+            - state C(latest) always triggers C(update_cache=yes)
+            - state C(rebuild) implies cast of all specified spells, not only
+              those existed before
+        required: false
+        choices: ["present", "latest", "absent", "cast", "dispelled", "rebuild"]
+        default: "present"
+
+    depends:
+        description:
+            - Comma-separated list of _optional_ dependencies to build a spell
+              (or make sure it is built) with; use +/- in front of dependency
+              to turn it on/off ('+' is optional though)
+            - this option is ignored if C(name) parameter is equal to '*' or
+              contains more than one spell
+            - providers must be supplied in the form recognized by Sorcery, e.g.
+              'openssl(SSL)'
+        required: false
+        default: null
+
+    update:
+        description:
+            - Whether or not to update sorcery scripts at the very first stage
+        required: false
+        choices: ["yes", "no"]
+        default: "no"
+
+    update_cache:
+        description:
+            - Whether or not to update grimoire collection before casting spells
+        required: false
+        aliases: ["update_codex"]
+        choices: ["yes", "no"]
+        default: "no"
+
+    cache_valid_time:
+        description:
+            - Time in seconds to invalidate grimoire collection on update
+            - especially useful for SCM and rsync grimoires
+            - makes sense only in pair with C(update_cache)
+        required: false
+        default: null
+'''
+
+
+EXAMPLES = '''
```

Comment:
Can you please rewrite the examples with a 
`name:` and multi line statements using `key: value`
We are in the process of standardising all the modules to use that format.